### DISCUSSION TITLE
请问为何已经完成了支付，但是日期还是没有更新，显示即将过期。

### DISCUSSION THREAD

**Discussion by Thomasyu-star** [getlantern/lantern-forum-cn#664] [Q&A]
用户名：15901681500@163.com

**Comment by Thomasyu-star** [comment]
![新建位图图像](https://github.com/getlantern/lantern-forum-cn/assets/174389722/73608c16-8f4f-4ca0-bf26-33bb1a59838e)

**Comment by MaxMKMax** [comment]
@Thomasyu-star 您好. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

**Comment by Thomasyu-star** [comment]
你好，这个问题还是无法解决，可以安排退款吗。发自我的iPhone------------------ 原始邮件 ------------------发件人: Mike K  ***@***.***>发送时间: 2024年7月2日 14:41收件人: getlantern/lantern-forum-cn ***@***.***>抄送: Thomasyu-star ***@***.***>, Mention ***@***.***>主题: Re: [getlantern/lantern-forum-cn] 请问为何已经完成了支付，但是日期还是没有更新，显示即将过期。 (Discussion #664)
@Thomasyu-star 您好. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱.
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/getlantern/lantern-forum-cn/discussions/664#discussioncomment-9932664",
"url": "https://github.com/getlantern/lantern-forum-cn/discussions/664#discussioncomment-9932664",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by Thomasyu-star** [comment]
2024.07.04 问题没有解决，请安排退款，谢谢。发自我的iPhone------------------ 原始邮件 ------------------发件人: Mike K  ***@***.***>发送时间: 2024年7月2日 14:41收件人: getlantern/lantern-forum-cn ***@***.***>抄送: Thomasyu-star ***@***.***>, Mention ***@***.***>主题: Re: [getlantern/lantern-forum-cn] 请问为何已经完成了支付，但是日期还是没有更新，显示即将过期。 (Discussion #664)
@Thomasyu-star 您好. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱.
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/getlantern/lantern-forum-cn/discussions/664#discussioncomment-9932664",
"url": "https://github.com/getlantern/lantern-forum-cn/discussions/664#discussioncomment-9932664",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by Thomasyu-star** [comment]
2024.07.05 无法使用，请安排退款。发自我的iPhone------------------ 原始邮件 ------------------发件人: Mike K  ***@***.***>发送时间: 2024年7月2日 14:41收件人: getlantern/lantern-forum-cn ***@***.***>抄送: Thomasyu-star ***@***.***>, Mention ***@***.***>主题: Re: [getlantern/lantern-forum-cn] 请问为何已经完成了支付，但是日期还是没有更新，显示即将过期。 (Discussion #664)
@Thomasyu-star 您好. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱.
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/getlantern/lantern-forum-cn/discussions/664#discussioncomment-9932664",
"url": "https://github.com/getlantern/lantern-forum-cn/discussions/664#discussioncomment-9932664",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by MaxMKMax** [comment]
@Thomasyu-star 
您好. 您放心, 只要您提交问题在链接, 我们的相关部门会帮您处理. 谢谢您支持蓝灯.

**Comment by Thomasyu-star** [comment]
2024/7/7 问题尚未解决，既五退款，也无续期。发自我的iPhone------------------ 原始邮件 ------------------发件人: Mike K  ***@***.***>发送时间: 2024年7月6日 00:08收件人: getlantern/lantern-forum-cn ***@***.***>抄送: Thomasyu-star ***@***.***>, Mention ***@***.***>主题: Re: [getlantern/lantern-forum-cn] 请问为何已经完成了支付，但是日期还是没有更新，显示即将过期。 (Discussion #664)
@Thomasyu-star
您好. 您放心, 我们的相关部门会帮您处理. 谢谢您支持蓝灯.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/getlantern/lantern-forum-cn/discussions/664#discussioncomment-9969996",
"url": "https://github.com/getlantern/lantern-forum-cn/discussions/664#discussioncomment-9969996",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by Thomasyu-star** [comment]
2024.07.08  本日无进展，请尽快安排退款。发自我的iPhone------------------ 原始邮件 ------------------发件人: Mike K  ***@***.***>发送时间: 2024年7月6日 00:08收件人: getlantern/lantern-forum-cn ***@***.***>抄送: Thomasyu-star ***@***.***>, Mention ***@***.***>主题: Re: [getlantern/lantern-forum-cn] 请问为何已经完成了支付，但是日期还是没有更新，显示即将过期。 (Discussion #664)
@Thomasyu-star
您好. 您放心, 我们的相关部门会帮您处理. 谢谢您支持蓝灯.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/getlantern/lantern-forum-cn/discussions/664#discussioncomment-9969996",
"url": "https://github.com/getlantern/lantern-forum-cn/discussions/664#discussioncomment-9969996",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Reply by MaxMKMax** [reply]
@Thomasyu-star 您需要退款的话请您再发申请链接退款. 谢谢您.

**Comment by Thomasyu-star** [comment]
日期已经更新，不用退款了，谢谢。发自我的iPhone------------------ 原始邮件 ------------------发件人: Mike K  ***@***.***>发送时间: 2024年7月9日 00:40收件人: getlantern/lantern-forum-cn ***@***.***>抄送: Thomasyu-star ***@***.***>, Mention ***@***.***>主题: Re: [getlantern/lantern-forum-cn] 请问为何已经完成了支付，但是日期还是没有更新，显示即将过期。 (Discussion #664)
@Thomasyu-star 您需要退款的话请您再发申请链接退款. 谢谢您.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/getlantern/lantern-forum-cn/discussions/664#discussioncomment-9989680",
"url": "https://github.com/getlantern/lantern-forum-cn/discussions/664#discussioncomment-9989680",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Reply by MaxMKMax** [reply]
@Thomasyu-star 好的. 没问题. 我已经帮您通知客服不弄退款了. 谢谢您支持蓝灯.
